Study_Session_2/README.md

```diff
@@ -0,0 +1,25 @@
+# Session 2
+
+**Date of session** : 23rd March 2021
+
+Objectives of the session:
+

```

Comment:
Can you elaborate on the things mentioned here? I mean just a small paragraph containing the summary of what we covered would look nice.

Reply:
Sure, I am adding it.